src/Compilers/Test/Utilities/CSharp/NullableAttributesVisitor.cs

```diff
@@ -46,19 +50,40 @@ public override void VisitNamespace(NamespaceSymbol @namespace)
 
         public override void VisitNamedType(NamedTypeSymbol type)
         {
+            var previousContext = _nullableContext;
+            _nullableContext = GetNullableContextAttribute(type.GetAttributes()) ?? _nullableContext;
+
             ReportSymbol(type);
             VisitList(type.TypeParameters);
-            VisitList(type.GetMembers());
+
+            foreach (var member in type.GetMembers())
+            {
+                // Skip accessors since those are covered by associated symbol.
+                if (member.IsAccessor()) continue;
+                Visit(member);
+            }
+
+            _nullableContext = previousContext;
         }
 
         public override void VisitMethod(MethodSymbol method)
         {
-            // Skip accessors since those are covered by associated symbol.
-            if (method.IsAccessor()) return;
+            var previousContext = _nullableContext;
+            _nullableContext = GetNullableContextAttribute(method.GetAttributes()) ?? _nullableContext;
 
             ReportSymbol(method);
             VisitList(method.TypeParameters);
             VisitList(method.Parameters);
+
+            _nullableContext = previousContext;
+        }
+
+        public override void VisitProperty(PropertySymbol property)
+        {
+            ReportSymbol(property);
+            VisitList(property.Parameters);
```

Comment:
>VisitList(property.Parameters); [](start = 12, length = 31)

Are we emitting any attributes for parameters of properties?

Reply:
Yes.

---
In reply to: [296908823](https://github.com/dotnet/roslyn/pull/36152#discussion_r296908823) [](ancestors = 296908823)

Reply:
> Yes.

I don't think so. I believe properties do not have parameters in metadata. 

---
In reply to: [297293892](https://github.com/dotnet/roslyn/pull/36152#discussion_r297293892) [](ancestors = 297293892,296908823)